ext/drmgr/drmgr.h

```diff
@@ -452,6 +465,53 @@ drmgr_unregister_bb_instrumentation_ex_event(drmgr_app2app_ex_cb_t app2app_func,
                                              drmgr_insertion_cb_t insertion_func,
                                              drmgr_ilist_ex_cb_t instru2instru_func);
 
+DR_EXPORT
+/**
+ * Registers callback functions for the third
+ * instrumentation stage: instrumentation
+ * insertion.  drmgr will call \p func for each instruction with the
+ * specific opcode \p opcode.
+ *
+ * More than one call-back function can be mapped to the same opcode. Their
+ * execution sequence is determined by their priority \p priority if set. Ordering
+ * based on priority is also taken in to account with respect to insert bb events.
+ *
+ * Since this call-back is triggered during insert instrumentation,
+ * same usage rules apply. The call-back is allowed to insert meta
+ * instructions only immediately prior to the passed-in instruction.
+ * New non-meta instructions cannot be inserted.
+ *
+ * \return false upon failure.
+ *
+ * @param[in]  func  The opcode insertion callback to be called for the third
+ * stage for a specific opcode instruction. Cannot be NULL.
+ * @param[in]  opcode          The opcode to associate with the insertion callback.
+ * @param[in]  priority        Specifies the relative ordering of both callbacks.
+ *                             Can be NULL, in which case a default priority is used.
+ *
+ * \note It is possible that this call-back will be triggered for meta instructions.
+ * Therefore, we recommend that the call-back checks for meta instructions
+ * (and ignore them, typically).
+ */
+bool
+drmgr_register_opcode_instrumentation_event(drmgr_opcode_insertion_cb_t func, int opcode,
+                                            drmgr_priority_t *priority);
+
+DR_EXPORT
+/**
+ * Unregisters the opcode-specific callback that
+ * was registered via drmgr_register_opcode_instrumentation_event().
+ *
+ * \return true if unregistration is successful and false if it is not
+ * (e.g., \p func was not registered for the passed opcode \p opcode).
+ *
+ * The recommendations for #dr_unregister_bb_event() about when it
+ * is safe to unregister apply here as well.
```

Comment:
s/#// (the empty parens make the link for functions: http://www.doxygen.nl/manual/autolink.html)

Reply:
Done - I fixed other docs as well.